### ISSUE TITLE
move everything from /usr/local to /usr

### ISSUE THREAD

**Issue by taukakao** [Vanilla-OS/desktop-image#175]
/usr/local is reserved for the system admin. Every software delivered by us needs to be in /usr. 
https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s09.html

This is also required for https://github.com/Vanilla-OS/ABRoot/issues/330

Currently installed software in /usr/local is:
- abroot-rollback-notifier
- adwdialog
- apx-gui
- vanilla-sideload
- vanilla-tour
- vanilla-updates-utility

**Comment by jardon** [comment]
Once https://github.com/Vanilla-OS/Vib/pull/108 lands i can move the meson-related items to the right prefix (apx-gui, vanilla-updates-utility, abroot-rollback-notifier)

**Comment by taukakao** [comment]
This doesn't seem to have worked.
The latest dev image still contains "abroot-rollback-notifier" "apx-gui" and "vanilla-updates-utility" in /usr/local/bin

**Comment by jardon** [comment]
i noticed this today.  ill look at it again

**Comment by jardon** [comment]
actually, this isnt fixed because we're still building with vib `0.8.1` which is from August.  the feat for this was merged into vib in September.

**Comment by taukakao** [comment]
I see. I think it's fine to just wait for the next vib version.

**Comment by NN708** [comment]
Resolved in #265.
